bin/dimm_csc

```diff
@@ -21,8 +21,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-import asyncio
+from lsst.ts.dimm.dimm_csc import dimm_csc
 
-from lsst.ts.dimm.dimm_csc import DIMMCSC
-
-asyncio.run(DIMMCSC.amain(index=True))
+dimm_csc()
```

Comment:
I gently suggest renaming this file and function to `run_dimm_csc` or `run_dimm`, or at least adding a verb to the name. That way if you ever want to add a CSC commander you can use `command_dimm_csc` or `command_dimm`. And similarly if you find you want other command-line operations on the dimm (unlikely) you can pick an unambiguous name.

Reply:
Good suggestion. Done.